conf/composites.conf

```diff
@@ -53,7 +53,17 @@ composites {
     expression = "R_DKIM_NA & R_SPF_NA & DMARC_NA & ARC_NA";
     score = 1.0;
     policy = "remove_weight";
-    description = "Authenticating message via SPF/DKIM/DMARC/ARC not possible";
+    description = "Authenticating message via SPF/DKIM/DMARC/ARC not available";
+  }
+  AUTH_NA_OR_FAIL {
+    expression = "!(R_DKIM_NA & R_SPF_NA & DMARC_NA & ARC_NA) & (R_DKIM_NA | R_DKIM_TEMPFAIL | R_DKIM_PERMFAIL) & (R_SPF_NA | R_SPF_DNSFAIL) & DMARC_NA & (ARC_NA | ARC_DNSFAIL)";
```

Comment:
I see `R_DKIM_TEMPFAIL` twice in the second braces, it is not likely intended.

Reply:
Now I think of it, maybe `AUTH_FAIL` is a better description and it would be easier to write it as: `R_DKIM_TEMPFAIL | R_DKIM_PERMFAIL | R_SPF_DNSFAIL | ARC_DNSFAIL`.

Reply:
It is because recursive composites are not allowed. If it would, it says.

`!AUTH_NA & (R_DKIM_NA | R_DKIM_TEMPFAIL | R_DKIM_PERMFAIL) & (R_SPF_NA | R_SPF_DNSFAIL) & DMARC_NA & (ARC_NA | ARC_DNSFAIL)`.

So, this applies when AUTH_NA is not applied, but still no authentication method is succeeding. Therefore I called it AUTH_NA_OR_FAIL.

Reply:
I could simplify the name to `AUTH_FAIL`.